pages/server/database/influxdb.mdx

```diff
@@ -11,11 +11,115 @@
 
 To create a server, we suppose that you've already [created and linked an account into the panel](/getting-started).
 
-In that case, go to DBH server and run this command:
+In that case, go to DBH server and,
```

Comment:
One nitpick, but shouldn't this go from `In that case, go to DBH server and,` to `In that case, go to DBH server and run either of these commands:` for more clarity? This goes as well for the other files.

(I saw this PR from the github channel in the server and I just want to state my opinions here)

Reply:
ah fair